integration_tests/mockApis/applications.ts

```diff
@@ -48,18 +49,7 @@ export default {
       response: {
         status: 201,
         headers: { 'Content-Type': 'application/json;charset=UTF-8' },
-        body: `
-        {
-          "id": "{{request.pathSegments.[1]}}",
-          "person": ${JSON.stringify(args.application.person)},
-          "createdByProbationOfficerId": "${args.application.createdByUserId}",
-          "schemaVersion": "${args.application.schemaVersion}",
-          "createdAt": "${args.application.createdAt}",
-          "submittedAt": "${args.application.submittedAt}",
-          "data": {{{jsonPath request.body '$.data'}}}
-        }
-        `,
-        transformers: ['response-template'],
+        jsonBody: { ...args.application, type: 'CAS2' },
```

Comment:
I'm guessing that `jsonBody` rather than `body` invokes some inbuilt JSON parsing?

Reply:
yep exactly. I'll pop a link to the docs in the commit message https://wiremock.org/docs/stubbing/#specifying-the-response-body